### ISSUE TITLE
Pipeline to generate poster alt text

### ISSUE THREAD

**Issue by guites** [cumbucadev/cinemaempoa#242]
Poster alt text can only be generated through llms in the screening creation/editing screen, which
is accessed manually by the website administrator.

This doesn't scale and we can confirm that by the fact that we have a very small amount of
poster with descriptions.

In order to improve accessibility, we should regularly check for images with missing descriptions
and run them through the API for generation.
